### PR TITLE
Include a way to get request variables without sanatization

### PR DESCRIPTION
**Pull request by bordoni**
### 🎫 Ticket

[TEC-5283]

### 🗒️ Description

Resolves the problem where because we are using `tribe_get_request_var` it will strip all HTML.

### 📹 Artifact

https://i.bordoni.me/Zt3Zk5Bd

[TEC-5283]: https://stellarwp.atlassian.net/browse/TEC-5283?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dpanta94 — CHANGES_REQUESTED** [review]
The code is good. I would prefer seeing another method though `tribe_get_request_var_raw` which would be used inside `tribe_get_request_var`, rather than a new parameter.

In most cases where we want a get param, we want to use the `tribe_get_request_var_raw`, if what we want to get needs to be saved or displayed we need `tribe_get_request_var` :D

**Review by dpanta94 — APPROVED** [review]

File: src/functions/utils.php [hunk 1]
```diff
@@ -156,36 +156,95 @@
...
 		}
 
 		if ( isset( $_GET ) ) {
-			$requests[] = (array) $_GET;
+			$requests[] = (array) $_GET; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.NonceVerification.Recommended
 		}
 
 		if ( isset( $_POST ) ) {
-			$requests[] = (array) $_POST;
+			$requests[] = (array) $_POST; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.NonceVerification.Recommended
```
Comment (Camwyn) [inline src/functions/utils.php]: Just disable at the beginning of the function and re-enable at the end?

File: src/functions/utils.php [hunk 2]
```diff
@@ -156,36 +156,95 @@ function tribe_exit( $status = '' ) {
...
-			return $default;
+			return $default_value;
 		}
 
-		$unsafe = Tribe__Utils__Array::get_in_any( $requests, $var, $default );
-		return tribe_sanitize_deep( $unsafe );
+		$unsafe = Tribe__Utils__Array::get_in_any( $requests, $request_var, $default_value );
+
+		// Return the value as is.
+		return $unsafe;
```
Comment (Camwyn) [inline src/functions/utils.php:247]:
```suggestion
		// Return the value as is.
		return Tribe__Utils__Array::get_in_any( $requests, $request_var, $default_value );
```

**Review by Camwyn — CHANGES_REQUESTED** [review]
Small tweaks and maybe less phpcs noise?

**Review by dpanta94 — DISMISSED** [review]
The code looks good - Why EVA tests are failing though ?

**Pull request merged**